includes/TingSearchCqlDoctor.class.inc

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * @file
+ * Class TingSearchCqlDoctor
+ * An attempt to convert non cql search phrases to valid cql.
+ *
+ * examples not valid:
+ * portland (film) => portland and (film)
+ * henning mortensen (f. 1939) => henning and mortensen and "(f. 1939)"
+ * harry and (White night) = harry and "(White night)"
+ *
+ *
+ * valid cql stays the same eg.
+ * dkcclterm.sf=v and dkcclterm.uu=nt and (term.type=bog) not term.literaryForm=fiktion
+ */
+
+class TingSearchCqlDoctor {
+
+  private $cql_string;
+  private $pattern;
+  private $replace;
+  private static $replace_key = 10;
+
+  /**
+   * constructor,
+   * @param $string
+   * sets private member $cql_string with trimmed string
+   */
+  public function __construct($string) {
+    $this->cql_string = trim($string);
+    // handle reserved characters
```

Comment:
Comments should be complete sentences starting with capital letter and ending with full-stop.